extensions/wikia/ExactTargetUpdates/tasks/ExactTargetWikiTask.class.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Wikia\ExactTarget;
+
+use Wikia\Tasks\Tasks\BaseTask;
+use Wikia\Util\Assert;
+
+class ExactTargetWikiTask extends BaseTask {
+
+	private $client;
+
+	const STATUS_OK = 'OK';
+
+	public function __construct( $client = null ) {
+		$this->client = $client;
+	}
+
+	/**
+	 * Update wiki or create if doesn't exist
+	 * @param int $wikiId
+	 * @return string
+	 * @throws \Wikia\Util\AssertionException
+	 */
+	public function updateWiki( $wikiId ) {
+		Assert::true( !empty( $wikiId ), 'Wiki ID missing' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Wiki ID missing" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1192)